### ISSUE TITLE
Node randomly falling out of sync

### ISSUE THREAD

**Issue by Crypto2** [binance-chain/node-binary#286]
Two times this week, both shortly after midnight UTC (not sure if that is relevant), our node has stopped syncing. We are running 0.8.2. All that is in the log is repeated messages like this:

I[2021-05-12|21:46:21.788] Ignoring message received during fastSync/hotSync module=consensus msg="[Vote Vote{1:14CFCE69B645 162544271/00/2(Precommit) 4E501BCB3657 80DE64E248B2 @ 2021-05-12T00:36:48.220062479Z}]"

162742128 was the last block it got to before it stopped, I don't see any obvious errors in the log or anything just lines like above and the normal comings and goings of peers.

**Comment by BillyFnh** [comment]
Hi, any updates on this?

**Comment by Crypto2** [comment]
Nope, I don't remember if it fixed itself or what happened though at this point :(
